src/uct/ib/base/ib_md.c

```diff
@@ -1645,11 +1730,10 @@ static ucs_status_t uct_ib_verbs_md_open(struct ibv_device *ibv_device,
 static uct_ib_md_ops_t uct_ib_verbs_md_ops = {
     .open                = uct_ib_verbs_md_open,
     .cleanup             = (uct_ib_md_cleanup_func_t)ucs_empty_function,
-    .memh_struct_size    = sizeof(uct_ib_verbs_mem_t),
     .reg_key             = uct_ib_verbs_reg_key,
     .dereg_key           = uct_ib_verbs_dereg_key,
-    .reg_atomic_key      = (uct_ib_md_reg_atomic_key_func_t)ucs_empty_function_return_unsupported,
-    .dereg_atomic_key    = (uct_ib_md_dereg_atomic_key_func_t)ucs_empty_function_return_unsupported,
+    .reg_atomic_key      = uct_ib_verbs_reg_atomic_key,
+    .dereg_atomic_key    = (uct_ib_md_dereg_atomic_key_func_t)ucs_empty_function_return_success,
```

Comment:
why verbs need uct_ib_verbs_reg_atomic_key? it doesn't really do anything, since dereg_atomic_key is empty

Reply:
to store strict order rkey into atomic_rkey if have

Reply:
ok